src/factories/workflow-specific/PIM_WorkflowFactory_v1.sol

```diff
@@ -125,20 +125,15 @@ contract PIM_WorkflowFactory_v1 is
         if (PIMConfig.isRenouncedIssuanceToken) {
             _transferTokenOwnership(issuanceToken, address(0));
         } else {
-            _transferTokenOwnership(
-                issuanceToken, PIMConfig.issuanceTokenParams.initialAdmin
-            );
+            _transferTokenOwnership(issuanceToken, PIMConfig.admin);
         }
 
-        // if renounced workflow flag is set factory keeps admin rights over workflow, else transfer admin rights to initial admin
+        // if renounced workflow flag is set factory keeps admin rights over workflow, else transfer admin rights to deployer
```

Comment:
is it the `deployer` technically? Isn't it a value that the deployer may set, but doesn't need to be the deployer? Or is this just because the UI will set this to the same address as the deployer normally?

Reply:
outdated docs, it should be the "admin specified in the params", will change.